Task2/dashboard.js

```diff
@@ -0,0 +1,246 @@
+// fetch('https://reqres.in/api/users')
+// .then(res=>res.json())
+// .then(data=>console.log(data))
+
+// let email=localStorage.getItem('email')?localStorage.getItem('email'):''
+// console.log(email);
+// if(email=='')
+// {
+//     alert('u need to login first');
+//     window.location.href="index.html";
+// }
+
+const root = document.getElementById("root");
+const form = document.getElementById("Form");
+const Post = document.getElementById("btn");
+// const list=document.createDocumentFragment();
+const url = "https://reqres.in/api/users";
+let users = new Array();
+console.log(users);
+async function Get() {
+  try {
+    const res = await fetch(url);
+    //console.log(res);
+    const data = await res.json();
+    //console.log(data);
+    let persons = data;
+    //console.log(data);
+    // console.log(persons.data)
+    persons.data.map((values) => {
+      users.push({
+        name: values.first_name,
+        email: values.email,
+        img: values.avatar,
+      });
+    });
+    users.map((ele) => {
+      displayFn(ele);
+    });
+  } catch (err) {
+    console.log(err);
+  }
+}
+Get();
+
+// diplaying the data
+const displayFn = (ele) => {
+  //console.log(ele);
+  const card = document.createElement("div");
+  card.innerHTML = `<div class="container ">
+            <div class="row">
+            <div class="col-sm-6">
+                    <img class="card-img-top" src="${ele.img}" altSrc="img">
+                    <div class="card-body">
+                    <h5 class="card-title">${ele.name}</h5>
+                    <h5 class="card-title">${ele.email}</h5>
+                    </div>
+                </div>
+            </div>
+          </div>`;
+  root.appendChild(card);
+};
+
+//funtion to logout
+// function Logout() {
+//   // window.localStorage.clear();
+//   window.location.reload(true);
+//   window.location.replace("index.html");
+// }
+let name=localStorage.getItem('name')?localStorage.getItem('name'):''
+console.log(name);
+if(name=='')
+{
+  alert('U need to login first');
+  window.location.href="index.html";
+}
+function Logout()
+{
+  localStorage.removeItem('name');
+  localStorage.removeItem('email');
+  window.location.href="index.html";
+}
+//function to go to create new user form
+function createPost() {
+  console.log(form);
+  //window.location.href='createPost.html'
+  if (form.style.display == "none") {
+    form.style.display = "block";
+    Post.style.display = "none";
+  }
+  
+}
+
+//function to post the data
+async function create() {
+  try {
+    const name = document.getElementById("name").value;
+    const job = document.getElementById("job").value;
+    // const img = new Image();
+    // img.src = "./png.png";
+      const res = await fetch(url, {
+        method: "POST",
+        body: JSON.stringify({
+          name: name,
+          job: job,
+        }),
+        headers: { "Content-type": "application/json;charset=utf-8" },
+      });
+      const data = await res.json();
```

Comment:
Follow the changes suggested above in the `Get()` method here as well.

@Poorvi-3pillar